<launch>
    
    <node pkg="finderbot" name="mini_mapping" type="mini_mapping_node" output="screen" />
   
    <node pkg="gazebo_ros" name="gazebo_launch" type="gazebo" args="/home/jon/catkin_ws/src/finderbot/FinderBot_Gazebo/bot_in_a_box.world"/>

    <node pkg="finderbot" name="transformer" type="transform_node" />

</launch>